src/main.cpp

```diff
@@ -44,17 +45,16 @@ void update_signal(int &key_input) {
             if (shoot::throttle_code == ZERO_THROTTLE) {
                 printf("Throttle is zero\n");
             } else {
-                shoot::throttle_code -= 50;
+                shoot::throttle_code = MIN(
```

Comment:
should be `MAX`. Also, (curious), where is `MIN` defined?

Reply:
These are macros defined in `pico_platform`. See this [link](https://raspberrypi.github.io/pico-sdk-doxygen/group__pico__platform.html)